opm/autodiff/BlackoilModelBase_impl.hpp

```diff
@@ -670,6 +672,13 @@ typedef Eigen::Array<double,
                     sd_.soMax = fluid_.satOilMax();
                     fluid_.getGasOilHystParams(sd_.krnswdc_go, sd_.pcswmdc_go, cells_);
                     fluid_.getOilWaterHystParams(sd_.krnswdc_ow, sd_.pcswmdc_ow, cells_);
+
+                    sd_.Pb = fluid_.bubblePointPressure(cells_,
```

Comment:
I got a feeling that doing this unconditionally would cause an exception for water-oil cases, but when testing it I ran into other problems with two-phase runs. I think there are a few bugs there we must fix before the release. After this release we *must* add a two-phase test case!